src/unix/fs.c

```diff
@@ -231,7 +231,29 @@ static ssize_t uv__fs_futime(uv_fs_t* req) {
 
 
 static ssize_t uv__fs_mkdtemp(uv_fs_t* req) {
-  return mkdtemp((char*) req->path) ? 0 : -1;
+  char *path = (char*) req->path;
+
+#if defined(__MVS__)
+  /* There is no mkdtemp. So instead use mktemp to generate a
```

Comment:
The code checks for errno==EEXIST in that case and then retry the mkstemp.
I just realized that i need to reset the path template to have "XXXXXX" at the end for that to work.
I will put in that fix.